ledger/participant-integration-api/src/main/scala/platform/store/backend/localstore/IdentityProviderCheckStorageBackendImpl.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.daml.platform.store.backend.localstore
+
+import anorm.RowParser
+import anorm.SqlParser.int
+import com.daml.ledger.api.domain.IdentityProviderId
+import com.daml.platform.store.backend.common.SimpleSqlAsVectorOf._
+
+import java.sql.Connection
+
+object IdentityProviderCheckStorageBackendImpl extends IdentityProviderCheckStorageBackend {
+  private val IntParser0: RowParser[Int] =
+    int("dummy") map { i => i }
+
+  def idpConfigByIdExists(id: IdentityProviderId.Id)(connection: Connection): Boolean = {
+    import com.daml.platform.store.backend.common.ComposableQuery.SqlStringInterpolation
+    val res: Seq[_] =
+      SQL"""
+         SELECT 1 AS dummy
+         FROM participant_identity_provider_config idpcfg
```

Comment:
```suggestion
         FROM participant_identity_provider_config t
```
?
`idpcfg` is hard to parse